tests/hpc/barrier_init.pm

```diff
@@ -0,0 +1,67 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Initialization of barriers for HPC multimachine tests
+# Maintainer: Petr Cervinka <pcervinka@suse.com>
+
+use base 'opensusebasetest';
+use strict;
+use testapi;
+use lockapi;
+use utils;
+
+sub run {
+    # Get number of nodes
+    my $nodes = get_required_var("CLUSTER_NODES");
+
+    # Initialize barriers
+    if (check_var("HPC", "slurm")) {
```

Comment:
Can I find somewhere info about what's slurm/mrsh/munge and why they use different barriers?

Reply:
slurm/mrsh/munge are packages from sle hpc module. They use different barriers, because they have different test scenarios.